packages/ng-form/src/lib/components/form/form.component.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges
+} from '@angular/core'
+import { FormGroup } from '@angular/forms'
+import { setFormEnabled } from '../../utils/form'
+
+@Component({
+  selector: 'ng-form-form',
```

Comment:
I don't imagine being happy whilst using this component in my templates:

`<ng-form-form>` 👿